ssl/s3_lib.c

```diff
@@ -842,8 +842,71 @@ static SSL_CIPHER ssl3_ciphers[] = {
      SSL_AES128GCM,
      SSL_AEAD,
      TLS1_3_VERSION, TLS1_3_VERSION,
+     SSL_kANY,
+     SSL_aANY,
+     SSL_HIGH,
+     SSL_HANDSHAKE_MAC_SHA256 | TLS1_PRF_SHA256,
```

Comment:
Ah I see we're using the handshake MD for that instead. I'll remove that PRF bit.